css/index.css

```diff
@@ -64,10 +77,45 @@ h1, h2, h3, h4, h5 {
 
 p {
     line-height: 1.4;
```

Comment:
flex-direction: row is the default when you use display: flex; you do not need to specify it if you don't want to.